spec/kafka_spec.rb

```diff
@@ -9,6 +9,7 @@
         node.default['hadoop']['mapred_site']['mapreduce.framework.name'] = 'yarn'
         stub_command(/update-alternatives --display /).and_return(false)
         stub_command(/test -L /).and_return(false)
+        stub_command(/getent/).and_return(false)
```

Comment:
OK. Figured it out. I originally had `create_cdap_user` as true by default, which is what prompted this. I just removed them and verified and tests still pass.